docs/editors/emacs.md

```diff
@@ -125,6 +125,47 @@ usage with Emacs.
 
 ```
 
+## LSP Tips
+
+### Show navigable stack trace
+
+You can annotate your stack trace with code lenses (which requires the
+following bit of configuration mentioned earlier: `(lsp-mode . lsp-lens-mode)`). 
+These allow you to run actions from your code.
+
+One of these actions allow you to navigate your stack trace.
+
+You can annotate any stack trace that lives in your `lsp` project by
+marking it with your region and using `M-x lsp-metals-analyze-stacktrace`. 
```

Comment:
So this part may be a bit confusing since one might wonder what it means to "live" in your lsp project. I'm not 100% sure how it works in emacs, but I know in the other clients as long as you have the stacktrace copied in your local buffer, when you execute the `analyze-stacktrace` command, it pulls it from the buffer and then gives you the new one with code lenses. So it might be better to clarify that here if that is the case.

Reply:
thanks for the feedback, I have tried to make that clearer